modules/reactor/src/meshgenerators/PatternedCartesianMeshGenerator.C

```diff
@@ -118,6 +118,11 @@ PatternedCartesianMeshGenerator::validParams()
       "interface_boundary_id_shift_pattern",
       "User-defined shift values for each pattern cell. A double-indexed array starting with the "
       "upper-left corner.");
+  MooseEnum quad_elem_type("QUAD4 QUAD8 QUAD9", "QUAD4");
+  params.addParam<MooseEnum>(
+      "boundary_element_type",
```

Comment:
"boundary" could be a confusing term here as it is used with "element". Maybe something like "boundary_region_element_type" or "quad_element_type_in_boundary_region"?

Reply:
renamed to `boundary_region_element_type`